plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -1179,6 +1179,7 @@ protected void doTestMismatchSchemaTable(
         }
     }
 
+    @SuppressWarnings("UnnecessaryOptionalGet")
```

Comment:
same here -- is the errorprone wrong?
at least add a comment, but this may indicate a bigger problem

Reply:
In this case it's wrong (false positive). I'll leave a comment here

Reply:
it's an obvious but in the check.
it realized you're calling `xx.getDiscretePredicates().ifPresent()` and then `xx.getDiscretePredicates().get()`, but did NOT realize the `xx` part is not the same

please report to them and put link in the comment

Reply:
https://github.com/google/error-prone/issues/2101